src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,135 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final String SCAN_INDEX_MESSAGE = "Invalid index";
+    private static final String INDEX_EQUALS_SIZE = "Index equals with size";
+    private Object[] elementArray;
+    private int size;
+
+    public ArrayList() {
+        this.elementArray = new Object[DEFAULT_CAPACITY];
+    }
+
+    public Object[] grow() {
```

Comment:
```suggestion
    private Object[] grow() {
```
![image](https://user-images.githubusercontent.com/20326858/206918962-30a5041b-e5ef-453e-8cad-138a88c953c3.png)

Reply:
Метод нужно переставить после публичных, а не менять модификатор доступа.